### PR TITLE
Issue 4 et l

### PR DESCRIPTION
**Pull request by ethanteague**

### CODE REVIEW

File: docroot/css/app.scss [hunk 1]
```diff
@@ -30,6 +30,24 @@
...
   }
 }
+#map-canvas {
+  height: 500px;
+  width:960px;
+  margin: 0px;
+  padding: 0px
+}
+img[src="public/images/circle.gif"] {
+  background:#205493;
```
Comment (nerdstein) [inline docroot/css/app.scss:40]: Just curious, but doesn't this background color change by the provider type? would it make sense to add classes for these images?

File: docroot/public/js/map.js [hunk 2]
```diff
@@ -40,20 +40,72 @@ function renderMap() {
...
-
-      var infoWindow = new google.maps.InfoWindow;
-      google.maps.event.addListener(marker, 'click', function () {
-        infoWindow.setContent(html);
-        infoWindow.open(map, this);
-      });
+      /**
+       * ProviderType Legend
+       * 1) ?
+       * 2) Licensed Slot Contractor
```
Comment (nerdstein) [inline docroot/public/js/map.js:46]: I don't think these are the correct provider types.... Please see https://trello.com/c/hnwtGVCc/37-as-a-parent-i-want-better-names-for-the-provider-types-that-mean-something-to-me

**Review by nerdstein — CHANGES_REQUESTED** [review]

File: docroot/css/app.scss (hunk 1 above)
Comment (ethanteague) [reply docroot/css/app.scss:40]: Normally yes, but markers don't allow for adding classes - we would have to add a google map overlay to go this way (tried this approach, unsuccessfully).

File: docroot/public/js/map.js (hunk 2 above)
Comment (ethanteague) [reply docroot/public/js/map.js:46]: Fixing now.

File: docroot/js/map.js [hunk 3]
```diff
@@ -40,20 +40,77 @@ function renderMap() {
...
-
-      var infoWindow = new google.maps.InfoWindow;
-      google.maps.event.addListener(marker, 'click', function () {
-        infoWindow.setContent(html);
-        infoWindow.open(map, this);
-      });
+      /**
+       * ProviderType Legend
+       * 1) ?
+       * 2) Licensed Slot Contractor
```
Comment (nerdstein) [inline docroot/js/map.js:46]: Is Slot Contractor still used?

File: docroot/public/js/map.js (hunk 2 above)
Comment (nerdstein) [reply docroot/public/js/map.js:46]: Is this fixed?

**Pull request merged**